packages/@uppy/companion/test/__tests__/uploader.js

```diff
@@ -148,13 +149,12 @@ describe('uploader with tus protocol', () => {
       // emulate socket connection
       socketClient.connect(uploadToken)
       socketClient.onProgress(uploadToken, (message) => {
-        if (firstReceivedProgress == null) firstReceivedProgress = message.payload.bytesUploaded
+        if (firstReceivedProgress == null) firstReceivedProgress = message.payload
       })
       socketClient.onUploadSuccess(uploadToken, (message) => {
         try {
-          expect(message.payload.bytesTotal).toBe(fileContent.length)
```

Comment:
Why are we no longer expecting that?

Reply:
in this commit https://github.com/transloadit/uppy/commit/1b5c0d7d4b6ac2f387ac5699d587e281f6579afe#diff-04b10f44d32399240120a15fe7a11a634d53760af469bf3ef80411421a164055L149 I accidentally moved  that expect from onProgress. Turns out that onSuccess doesn't have any payload.bytesTotal, but we never knew because this test succeeded with a false positive due to a different problem (that this PR also fixes)